models/migrations/v16.go

```diff
@@ -1,25 +1,117 @@
-// Copyright 2016 Gitea. All rights reserved.
-// Use of this source code is governed by a MIT-style
-// license that can be found in the LICENSE file.
-
 package migrations
```

Comment:
it seems to be missing in master (https://github.com/go-gitea/gitea/blob/master/models/migrations/v16.go) and merged removed this, mine is v17.go